css-two-column-layout/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-full">
+        <h1>scale, speed, agility</h1>
+        <p>We've dedicated thousands of man hours to making
+          healthcare institutions in the Helium network run
+          10x more efficiently and deliver an unparalled quality
+          of care.</p>
+      </div>
+    </div>
+    <div class="row">
```

Comment:
as you are building a 2 column layout, I would recommend wrapping your `column fourth` classes with a `column half` class so that you have a true two column layout in the rows